ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentMetadata/DocumentExtras.js

```diff
@@ -0,0 +1,51 @@
+import { MetadataTable } from '@pages/backoffice';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Header, List, Segment } from 'semantic-ui-react';
+import capitalize from 'lodash/capitalize';
+import isEmpty from 'lodash/isEmpty';
+
+export class DocumentExtras extends Component {
+  renderConferenceInfo = () => {
+    const { conference_info } = this.props.document.metadata;
+    let rows = [];
+    for (const [key, val] of Object.entries(conference_info)) {
+      if (Array.isArray(val)) {
```

Comment:
Just to understand, we can have two different types of values for `conference_info`?  Array of objects as per document schema or string?

Reply:
no, there is a key in the conference_info which value is an array. But you made me wonder if it is a good way, maybe I should just specify all one by one